client/express-checkout/utils/checkPaymentMethodIsAvailable.js

```diff
@@ -15,11 +15,24 @@ import { getUPEConfig } from 'wcpay/utils/checkout';
 
 export const checkPaymentMethodIsAvailable = memoize(
 	( paymentMethod, cart, resolve ) => {
-		const root = ReactDOM.createRoot(
-			document.getElementById(
-				`express-checkout-check-availability-container-${ paymentMethod }`
-			)
-		);
+		// Create the DIV container on the fly
+		const containerlEl = document.createElement( 'div' );
```

Comment:
One minor fix here on the spelling, I believe should be `containerEl` (no extra l between container and El).

Reply:
done [c564b30](https://github.com/Automattic/woocommerce-payments/pull/9420/commits/c564b30d69abb7a2d62e5e5b9996fa1d49e05676)